upgrade/steps_helpers.go

```diff
@@ -404,24 +404,34 @@ func (g gitRepoRefs) sortedLabels(less func(string, string) bool) []string {
 	return labels
 }
 
-func latestRelease(ctx context.Context, repo string) (*semver.Version, error) {
-	resultBytes, err := exec.CommandContext(ctx, "gh", "repo", "view",
-		repo, "--json=latestRelease").Output()
-	if err != nil {
-		return nil, err
-	}
+var findCurrentMajorVersion = stepv2.Func21("Find Current Major Version",
+	func(ctx context.Context, repoOrg, repoName string) *semver.Version {
+		repoCurrentVersion := latestRelease(ctx, repoOrg+"/"+repoName)
+		stepv2.SetLabelf(ctx, "%d", repoCurrentVersion.Major())
+		return repoCurrentVersion
+	})
```

Comment:
complete nit because I have a weird brain: can we do this for readability?

```suggestion
	},
)
```